### DISCUSSION TITLE
A way to check Total AFK time?

### DISCUSSION THREAD

**Discussion by Gogolian** [ActivityWatch/activitywatch#535] [General]
Is there a way right now to check total afk time for a perpoid or a day?

**Comment by johan-bjareholt** [comment]
Nog right now, but the data is there so it would be possible and quite easy to implement.

**Comment by Gogolian** [comment]
I did found workaround. In Log files aw-watcher afk logs when you went afk and came back, so that's good enough for now.
(Leaving reply for anyone who's got the same problem)

**Comment by johan-bjareholt** [comment]
You can also see that data in the web-ui, both in the "timeline" view and in the "raw data" view.
